jar-jni/src/main/java/io/questdb/jar/jni/LibInfo.java

```diff
@@ -1,10 +1,10 @@
 package io.questdb.jar.jni;
 
 public class LibInfo {
```

Comment:
It's a customisation point.
For example in our own code we _always_ prefix our libs with "lib", even on Windows.

Reply:
I can drop all of this code, of course, if we migrate our own CMake binaries to standard platform conventions.

Reply:
Changing QuestDB's CMakeFile logic turned out to be pretty easy. It's a special case because we use MINGW. QuestDB's `oxidation` branch has it fixed now.

I'll be removing the `LibInfo` class altogether.